packages/adapter/README.md

```diff
@@ -4,14 +4,12 @@
 ![](https://img.shields.io/badge/yarn-%3E%3D1.17.0-orange.svg?style=flat-square)
 ![Discord](https://img.shields.io/discord/608204864593461248?color=blue&label=Discord&logo=discord)
 
-Metamask <> Polkadot snap adapter is used to inject [polkadot snap](https://github.com/chainsafe/metamask-snap-polkadot) as web3 provider. It lists snap inside `window.injectedWeb3["metamask-polkadot-snap"]` so it can be enabled using `@polkadot/extension-dapp` package.  
+Metamask <> Polkadot snap adapter is used to inject [polkadot snap](https://github.com/chainsafe/metamask-snap-polkadot) as web3 provider. It lists snap inside `window.injectedWeb3["Snap"]` so it can be enabled using `@polkadot/extension-dapp` package.  
```

Comment:
maybe we should allow users to configure injected key? Not sure Snap is most sane default, we could cut breaking change to adapter to set more sane default like metamask-polkadot-snap ?

Reply:
Sounds good